### DISCUSSION TITLE
bun-v0.6.9

### DISCUSSION THREAD

**Discussion by Jarred-Sumner** [oven-sh/bun#3300] [Announcements]
**[Read Bun v0.6.9's release notes on Bun's blog](https://bun.sh/blog/bun-v0.6.9)**

To install:
```bash
curl https://bun.sh/install | bash
# or you can use npm
# npm install -g bun
```

To upgrade:
```bash
bun upgrade
```

## What's Changed
* Fix wording and code example in docs for Bun.sleepSync by @paperluigis in https://github.com/oven-sh/bun/pull/3270
* Fixes #3250 by @Jarred-Sumner in https://github.com/oven-sh/bun/pull/3269
* Make benchmark run again by @lpinca in https://github.com/oven-sh/bun/pull/3276
* Make cold `bun install` use 2x less memory by @Jarred-Sumner in https://github.com/oven-sh/bun/pull/3271
* Support using `WTF::StringImpl` from Zig by @Jarred-Sumner in https://github.com/oven-sh/bun/pull/3279
* add --save argument to install by @kvakil in https://github.com/oven-sh/bun/pull/3277
* doc: minor fixes by @kvakil in https://github.com/oven-sh/bun/pull/3278
* typo in readline by @paperdave in https://github.com/oven-sh/bun/pull/3286
* Make mocks use FunctionPrototype by @Jarred-Sumner in https://github.com/oven-sh/bun/pull/3291
* handle unwrapping `require` in any expression by @dylan-conway in https://github.com/oven-sh/bun/pull/3292
* workaround quote escape issues for `bun run` by @alexlamsl in https://github.com/oven-sh/bun/pull/3290
* Fix crash in CJS by @Jarred-Sumner in https://github.com/oven-sh/bun/pull/3294

## New Contributors
* @paperluigis made their first contribution in https://github.com/oven-sh/bun/pull/3270
* @lpinca made their first contribution in https://github.com/oven-sh/bun/pull/3276
* @kvakil made their first contribution in https://github.com/oven-sh/bun/pull/3277

**Full Changelog**: https://github.com/oven-sh/bun/compare/bun-v0.6.8...bun-v0.6.9

<hr /><em>This discussion was created from the release <a href='https://github.com/oven-sh/bun/releases/tag/bun-v0.6.9'>bun-v0.6.9</a>.</em>

**Comment by zloirock** [comment]
![image](https://github.com/oven-sh/bun/assets/2213682/432fe59d-a9f0-46c4-bc2a-50ed60a90812)

**Comment by Jarred-Sumner** [comment]
Can you try again

**Reply by zloirock** [reply]
Now it works, thanks.

**Comment by dotzenith** [comment]
Nice

**Comment by Michael-Sjogren** [comment]
Nice
